go.sum

```diff
@@ -318,6 +318,7 @@ github.com/ckaznocha/protoc-gen-lint v0.2.4 h1:Vp7LuGORwOdAFEqgBEfLcZyKqdtzrN8lc
 github.com/ckaznocha/protoc-gen-lint v0.2.4/go.mod h1:x9chIbwX2yctbChCqkuASKiEMT2XzxNr8KMpvrwcIGE=
```

Comment:
hmm how come only the go.sum was updated?

Reply:
Due to this: https://app.circleci.com/pipelines/github/stackrox/scanner/26474/workflows/07119e8c-449c-4ace-b6fb-042d0c9e255f/jobs/262755/parallel-runs/0/steps/0-102

Reply:
That seems odd to me... Nothing has changed and the master branch is fine.

Reply:
ah perhaps it's because of the new direct dependency on `httputil` and other related packages